src/main/java/fi/aalto/cs/apluscourses/model/Exercise.java

```diff
@@ -28,6 +28,8 @@
 
   private final boolean submittable;
 
+  private boolean inGrading = false;
```

Comment:
`false` is the default, so can be omitted: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html